packages/lib/src/components/PayNow/PayNow.test.ts

```diff
@@ -21,4 +28,53 @@ describe('PayNow', () => {
             expect(paynow.render()).not.toBe(null);
         });
     });
+
+    test('should render mobile instructions', async () => {
+        // Mocks matchMedia to return 'matches: true' when checking (max-width: 1024px)
+        Object.defineProperty(window, 'matchMedia', {
+            writable: true,
+            value: jest.fn().mockImplementation(() => ({
+                matches: true
+            }))
+        });
+
+        jest.useFakeTimers();
+
+        const srPanel = mock<SRPanel>();
+        const resources = mock<Resources>();
+        resources.getImage.mockReturnValue((icon: string) => `https://checkout-adyen.com/${icon}`);
+
+        // @ts-ignore mockResolvedValue not inferred
+        checkPaymentStatus.mockResolvedValue({
+            payload: 'Ab02b4c0!BQABAgBLLk9evMb+rScNdE...',
+            resultCode: 'pending',
+            type: 'complete'
+        });
+
+        const paynow = new PayNow(global.core, {
+            loadingContext: 'checkoutshopper.com/',
+            modules: { resources, analytics: global.analytics, srPanel },
+            i18n: global.i18n,
+            paymentData: 'Ab02b4c0!BQABAgBH1f8hqfFxOvbfK..',
+            qrCodeImage: '',
+            paymentMethodType: 'paynow',
+            qrCodeData: '00020126580009SG...'
+        });
+
+        render(paynow.mount('body'));
```

Comment:
Wouldn't `render(paynow.render())` not work?

Reply:
It doesn't, there are errors about undefined props of SRPanel. I guess I need to properly mock the object for this test-case. I will try to fix it in the other PR